promscale/distributed-tracing.md

```diff
@@ -0,0 +1,288 @@
+# Tracing
+Promscale supports tracing using the [OpenTelemetry][opentelemetry] protocol
+(OTLP). Promscale supports the entire OpenTelemetry trace data model including
+spans, events, and links.
+
+You can also use Jaeger and Zipkin traces with the OpenTelemetry Collector. The
+OpenTelemetry Collector ingests data from Jaeger and Zipkin instrumentation,
+converts it to OpenTelemetry, and sends it to Promscale using OTLP.
+
+You can visualize traces in Promscale using Grafana and the Jaeger UI.
```

Comment:
I think that splits my clause. Perhaps this works better, if it needs a comma:

```suggestion
You can visualize traces in Promscale, using Grafana and the Jaeger UI.
```

Reply:
> I think that splits my clause. Perhaps this works better, if it needs a comma:

I suggested the Oxford comma because Grafana UI and Jaeger UI are two different products with different UIs. I am not sure if they have a combined UI. I may be wrong.

Reply:
It's not a series if there are only two items.

Reply:
> It's not a series if there are only two items.

Agreed!!!!!